docs/fidesops/docs/development/release_checklist.md

```diff
@@ -0,0 +1,12 @@
+# Release Checklist 
+
+
+## Documentation
+- [ ] Quickstart verified working and up-to-date
+- [ ] Tutorial verified working and up-to-date
+- [ ] Fidesdemo verified working and up-to-date
+- [ ] New/updated API endpoints described in the Guides
+- [ ] New/updated API endpoints included in the Postman collections
```

Comment:
Did we intentionally leave test data out of the JSON for the Postman collection? I've been meaning to make a task to go through and add a lot of that too